src/ex1/scripts/domScripts.js

```diff
@@ -0,0 +1,195 @@
+const taskInput = document.querySelector(".task-input");
+const addTaskButton = document.querySelector(".task-add-btn");
+const tasksList = document.querySelector(".tasks-list");
+const filterOptiopn = document.querySelector(".filter-tasks");
+const searchInput = document.querySelector(".search-input");
+
+addTaskButton.addEventListener("click", onAddTaskClick);
+tasksList.addEventListener("click", onTaskButtonsClick);
+filterOptiopn.addEventListener("click", onFilterOptionChange);
+searchInput.addEventListener("keyup", onSearchInputKeyUp);
+
+taskInput.addEventListener("keypress", function (event) {
+  if (event.key === "Enter") {
+    event.preventDefault();
+    addTaskButton.click();
+  }
+});
+
+function onAddTaskClick(e) {
+  const isOk = canProceed();
+  if (isOk) {
+    const taskDiv = document.createElement("div");
+    taskDiv.classList.add("task");
+
+    const gripLines = createGridLines();
+    taskDiv.appendChild(gripLines);
+
+    const taskContent = createTaskContent();
+    taskDiv.appendChild(taskContent);
+
+    const completeTaskButton = createCompleteBtn();
+    taskDiv.appendChild(completeTaskButton);
+
+    const removeTaskButton = createRemoveBtn();
+    taskDiv.appendChild(removeTaskButton);
```

Comment:
very nice and ordered code.
would suggest going even further and write a `createTaskElement` function that contains all of these - making your code even more neat